stories/module-analytics.stories.js

```diff
@@ -124,6 +128,51 @@ storiesOf( 'Analytics Module', module )
 			</WithTestRegistry>
 		);
 	} )
+	.add( 'Property Select including GA4 properties', () => {
+		// Have to import like with tests
+		enabledFeatures.add( 'ga4setup' );
```

Comment:
Due to the changes in `analytics/datastore/properties.js` to use `isFeatureEnabled`, I had to enable `ga4setup` in the story

But it only worked like this